packages/transactions/README.md

```diff
@@ -14,12 +14,321 @@
+
+This type represents a transaction for which a fee payer has been declared. A transaction must conform to this type to be landed on the network.
+
+### Functions
+
+#### `setTransactionFeePayer()`
+
+Given a base58-encoded address of a system account, this method will return a new transaction having the same type as the one supplied plus the `ITransactionWithFeePayer` type.
+
+```ts
+import { address } from '@solana/addresses';
+import { setTransactionFeePayer } from '@solana/transactions';
+
+const myAddress = address('mpngsFd4tmbUfzDYJayjKZwZcaR7aWb2793J6grLsGu');
+const txPaidByMe = setTransactionFeePayer(myAddress, tx);
+```
+
+## Defining a transaction's lifetime
+
+A signed transaction can be only be landed on the network if certain conditions are met:
+
+-   It includes the hash of a recent block
+-   It includes the value of an unused nonce known to the network
+
+These conditions define a transaction's lifetime, after which it can no longer be landed, even if signed.
+
+### Types
+
+#### `ITransactionWithBlockhashLifetime`
+
+This type represents a transaction whose lifetime is defined by the age of the blockhash it includes. Such a transaction can only be landed on the network if the current block height of the network is less than or equal to the value of `ITransactionWithBlockhashLifetime['lifetimeConstraint']['lastValidBlockHeight']`.
+
+#### `IDurableNonceTransaction`
+
+This type represents a transaction whose lifetime is defined by the value of a nonce it includes. Such a transaction can only be landed on the network if the nonce is known to the network and has not already been used to land a different transaction.
+
+#### `Blockhash`
+
+This type represents a string that is particularly known to be the base58-encoded value of a block.
+
+#### `Nonce`
+
+This type represents a string that is particularly known to be the base58-encoded value of a nonce.
+
+### Functions
+
+#### `setTransactionLifetimeUsingBlockhash()`
+
+Given a blockhash and the last block height at which that blockhash is considered usable to land transactions, this method will return a new transaction having the same type as the one supplied plus the `ITransactionWithBlockhashLifetime` type.
+
+```ts
+import { setTransactionLifetimeUsingBlockhash } from '@solana/transactions';
+
+const { value: latestBlockhash } = await rpc.getLatestBlockhash().send();
+const txWithBlockhashLifetime = setTransactionLifetimeUsingBlockhash(latestBlockhash, tx);
+```
+
+#### `setTransactionLifetimeUsingDurableNonce()`
+
+Given a nonce, the account where the value of the nonce is stored, and the address of the account authorized to consume that nonce, this method will return a new transaction having the same type as the one supplied plus the `IDurableNonceTransaction` type. In particular, this method _prepends_ an instruction to the transaction designed to consume (or &lsquo;advance&rsquo;) the nonce in the same transaction whose lifetime is defined by it.
```

Comment:
Is the quote encoding something I’m only seeing because I’m on my phone or..? (`&lsquo`)